src/tools/pcb_reader/navmesh_exporter.h

```diff
@@ -91,7 +91,7 @@ class NavmeshExporter
     auto end = std::chrono::high_resolution_clock::now();
     printf( "[Navmesh] Finished exporting %s in %lu ms\n",
       fileName.c_str(),
-      std::chrono::duration_cast< std::chrono::milliseconds >( end - start ).count() );
+      static_cast<unsigned long>(std::chrono::duration_cast< std::chrono::milliseconds >( end - start ).count() ));
```

Comment:
this static_cast is not ideal
prefer changing the format specifier %lu to match .count() type, for example %ld